src/Tizen.NUI/src/public/Theme/DefaultThemeCommon.cs

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#if !PROFILE_TV
```

Comment:
is this `PROFILE_TV` working?

Reply:
In my opinion, when we do the gbs build with vd profile, this constant hopefully need to work. I will verify this tomorrow in the office.

Reply:
`PROFILE_TV` works when buliding with VD repo.